kedro_serving/cli.py

```diff
@@ -21,6 +21,7 @@ def serving_commands():
 @click.option(
     "--pipeline",
     "-n",  # n for "name"
+    default="__default__",
```

Comment:
Actually I did not put a default value  for the ``pipeline`` argument of the ``serve`` command on purpose. I think many pipelines (including the default one) cannot be served with kedro-serving because they don't have a single input. In most ML projects, the ``__default__`` pipeline is a combination of training + predicting + computing some metrics, and there are very often other inputs than only the raw data. in your projects, do you often most times that the ``__default__`` can be served?